services/crypto_update.proto

```diff
@@ -129,4 +129,22 @@ message CryptoUpdateTransactionBody {
      * including implicit and explicit associations.
      */
     google.protobuf.Int32Value max_automatic_token_associations = 15;
+
+    /**
+     * The new alias public key that can be set for an AccountID. For existing accounts alias key can
```

Comment:
Not sure how an alias can be updated if it's only supposed to be useful for the creation of a new account.

Reply:
As discussed removed the crypto update changes from this release.